dav-controller/api/templates/verified_credentials.html

```diff
@@ -273,292 +134,406 @@
-          success.style.display = "none";
-          pending.style.display = "none";
-          failed.style.display = "none";
-          expired.style.display = "grid";
-
-          // Button elements
-          refreshButton.style.display = "flex";
-          deeplinkButton.style.display = "none";
-          scannedMask.style.display = "none";
-          newQRRequestButton.style.display = "none";
-
-          // Turn off the spinner
-          qrcode.classList.remove("pending");
-          revealedAttribs.style.display = "none";
-          break;
-        }
+    const getBrowser = () => {
+      let userAgent = navigator.userAgent || navigator.vendor;
+ 
+      if (/android/i.test(userAgent)) {
+        return "Android";
+      }
+ 
+      if (
+        /iPad|iPhone|iPod/.test(userAgent) ||
+        (/Macintosh/.test(userAgent) && "ontouchend" in document) ||
+        (navigator.platform === "MacIntel" && navigator.maxTouchPoints > 1) ||
+        (navigator.vendor && navigator.vendor.indexOf("Apple") > -1)
+      ) {
+        return "iOS";
+      }
+ 
+      return "unknown";
     };
-
-    document.getElementById("refresh-button").addEventListener("click", () => {
-      location.reload(true);
-    });
-    document.getElementById("new-request-btn").addEventListener("click", () => {
-      location.reload(true);
-    });
-    document.getElementById("deep-link-btn").addEventListener("click", () => {
-      window.open("{{deep_link_url}}", '_blank').focus();
-    });
-
-    let timer;
-
-    /**
-     * Check status through polling
-     * This functionality is mostly replaced by the websocket.
-     * However, it is still used to trigger the api to calculate
-     * if the proof has expired or not. The websocket in turn handles
-     * all other functionality.
-     */
-    const checkStatus = () => {
-      return new Promise((resolve, reject) => {
-        const url = `${window.location.origin}/age-verification/{{pid}}`;
-        fetch(url)
-          .then((res) => res.json())
-          .then((data) => {
-            if (data.status === 'success') {
-              let htmlContent = '<div class="revealed-attribs-table" id="revealed-attribs-table">';
-              for (const [key, value] of Object.entries(data.metadata.revealed_attributes)) {
-                if (key === 'picture') {
-                  htmlContent += `<img src="${value}" alt="Verified Picture" style="flex-shrink: 0"/>`;
-                } else {
-                  htmlContent += `<div>${key}: ${value}</div>`;
-                }
+ 
+    const app = createApp({
+      data() {
+        const mobileDevice = ref(
+          getBrowser() === "Android" || getBrowser() === "iOS" ? true : false
+        );
+        const failed = computed(() => state.current === "failed")
+ 
+        return {
+          mobileDevice,
+          state,
+          failed,
+          headerText: mobileDevice.value ? "Continue with:" : "Scan with a Digital Wallet",
```

Comment:
Can we change this to `Scan with BC Wallet`? It is the only supported wallet currently, so we may as well be explicit.